py/desispec/scripts/stdstars.py

```diff
@@ -189,6 +192,10 @@ def main(args, comm=None) :
         if rank == 0:
             log.info('GPU not available')
 
+    std_targetids = None
+    if args.std_targetids is not None:
+        std_targetids = [ int(_) for _ in args.std_targetids]
```

Comment:
Minor: targetids are int, so is there a specific reason to use `type=str` for argparse only to have to convert those strings to integers here?  i.e. I would suggest
```
parser.add_argument('--std-targetids', type=int, ... )
...
if args.std_targetids is not None:
    std_targetids = args.std_targetids...
```
I think that has the added benefit of if someone passes in a non-int, it stops at the level of argparsing rather than later in the code.

Reply:
That was an oversight on my part. I initially was planning to pass targetids as comma separated strings. I'll fix that